packages/docusaurus-plugin-content-blog/src/frontMatter.ts

```diff
@@ -13,6 +13,10 @@ import {
   FrontMatterTOCHeadingLevels,
   ContentVisibilitySchema,
 } from '@docusaurus/utils-validation';
+import {
+  FrontMatterAuthorErrorMessage,
+  FrontMatterLastUpdateErrorMessage,
+} from '@docusaurus/utils/lib/lastUpdateUtils';
```

Comment:
```suggestion
import {FrontMatterLastUpdateSchema} from '@docusaurus/utils-validation';
```

Also, we shouldn't import from `/lib/lastUpdateUtils`: this works atm but will be forbidden some day